blocked-edges/4.8.5.yaml

```diff
@@ -0,0 +1,3 @@
+to: 4.8.5
+from: 4\.7\..*
+# SDN is down and kube-apiserver cannot reach aggregated apiserver: no route to host, SDN Packet loss resulting in service unavailability.  https://bugzilla.redhat.com/show_bug.cgi?id=1987108
```

Comment:
All 4.8.2+ are known to be affected because of the bugs, so we also need to add this bug to previous blocked edges

Reply:
Added